examples/ex_joint_goal.py

```diff
@@ -33,6 +35,9 @@ def main():
             0.7853981633974483,
         ],
     )
+    node.declare_parameter("synchronous", True)
+    # If negative, don't cancel. Only used if synchronous is False
+    node.declare_parameter("cancel_after_secs", -1.0)
```

Comment:
I think that setting this parameter to `0.0` should also **not** cancel the execution. 
Would you consider changing this parameter to have an effect only for positive numbers and set the default to `0.0` (also in the other example)?